static/app/views/issueList/actions/actionSet.tsx

```diff
@@ -234,7 +255,7 @@ function ActionSet({
           showChevron: false,
           size: 'xs',
         }}
-        disabledKeys={disabledMenuItems}
+        disabledKeys={menuItems.filter(item => item.disabled).map(item => item.key)}
```

Comment:
Added `disabled` boolean to `menuItems` and deleted the array above to simplify things a bit